lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -302,10 +300,10 @@ export class ResponseHandler {
      * @param stateString
      */
     static async generateAuthenticationResult(
-        cryptoObj: ICrypto, 
+        cryptoObj: ICrypto,
         authority: Authority,
-        cacheRecord: CacheRecord, 
-        fromTokenCache: boolean, 
```

Comment:
There's a ton of little cleanup in here like addressing extra whitespace.  Suggest creating a separate PR just for code cleanup/refactoring in the future.  Makes the core functional changes a smaller PR which is often easier to review.

Reply:
Guessing the whitespace changes were done automatically by the IDE. FYI you can hide whitespace changes from the diff here: 
![image](https://user-images.githubusercontent.com/443409/173110425-2112e5b7-dcb6-4c7f-a380-e4aa3ebf8fef.png)

Reply:
Yep. It is my IDE setup. I do think however we need a the same IDE settings as a standard we can import and apply if we want to avoid this in PRs.